client/docker/confd/templates/parameters.yml.tmpl

```diff
@@ -20,7 +20,7 @@ parameters:
     log_path: /var/log/app/application.log
     ga:
         default: {{ getv "/ga/default" }}
-        gds: {{ getv "/ga/gds" }}
+        gds: ''
```

Comment:
Should this still be a blank string?

Reply:
It doesn't get used anywhere and it was causing the container to fall over under 1.4.0 for some reason. I tried just updating the variable it pulls from env file to `""` in the hope it would find it but it didn't and got fed up so blanked it out. Really I should find out why it's not picking it up when it's picking up `ga/default`. I'll do it. I know you're right! :)